app/views/users/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container-lg" style="margin-top: 20px">
+  <div class="row">
+    <div class="col">
+      <h2>
+        My Trips
+      </h2>
+    </div>
+    <div class="col" style="text-align: right">
```

Comment:
Bootstrap also has [text-allignment classes](https://getbootstrap.com/docs/4.6/utilities/text/#text-alignment) you can use instead of inline styles.